File: time-conversion.js
function timeConversion(s) {
    // Write your code here
    let [hour, minutes, seconds] = s.split(':')
    let meridiem = seconds.slice(-2)
    seconds = seconds.slice(0,2)
    
    if (meridiem === "AM" && hour === "12") {
        hour = "00"
    } else if (meridiem === "PM" && hour !== "12") {
        hour = parseInt(hour) + 12
    }
    
    return `${hour}:${minutes}:${seconds}`

}

File: time-conversion.py
def timeConversion(s):
    hour, minutes, seconds = s.split(":")
    print(seconds)
    meridiem = seconds[-2:]
    print(meridiem)
    seconds = seconds[:-2]
    
    if meridiem == "AM" and hour == "12":
        hour = "00"
    elif meridiem == "PM" and hour != "12":
        hour = int(hour) + 12
    
    return f'{hour}:{minutes}:{seconds}'
